.github/workflows/e2e-test.yml

```diff
@@ -58,6 +60,8 @@ jobs:
           ADYEN_API_KEY: ${{secrets.ADYEN_API_KEY}}
           ADYEN_CLIENT_KEY: ${{secrets.ADYEN_CLIENT_KEY}}
           ADYEN_MERCHANT: ${{secrets.ADYEN_MERCHANT}}
+          GOOGLE_USERNAME: ${{secrets.GOOGLE_USERNAME}}
+          GOOGLE_PASSWORD: ${{secrets.GOOGLE_PASSWORD}}
```

Comment:
Since we are not installing and enabling express Google Pay on this pipeline, do we still need to pass Google credentials?

Reply:
Actually we are enabling it, that's why it was required.

Reply:
My bad, fixed.